### PR TITLE
Create Rule “use-figma-dev-mode/rule”

### PR DESCRIPTION
**Pull request by bettybondoc**
[Product Backlog Item 91152](https://dev.azure.com/ssw/SSW.Design/_workitems/edit/91152): ✍️ SSW Rules - New Rule - Using Figma Dev Mode For Easy Handovers

### CODE REVIEW

File: rules/use-figma-dev-mode/rule.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+type: rule
+title: Do you use Figma Dev Mode to improve developer handovers?
+seoDescription: "Learn how Figma Dev Mode streamlines developer handovers by
+  providing a seamless bridge between design and development. type: rule"
```
Comment (danielmackay) [inline rules/use-figma-dev-mode/rule.md:5]:
```suggestion
seoDescription: Learn how Figma Dev Mode streamlines developer handovers by providing a seamless bridge between design and development. type: rule
```

File: rules/use-figma-dev-mode/rule.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    url: https://www.ssw.com.au/people/betty-bondoc/
+created: 2024-10-30T14:20:35.753Z
+guid: 83be089e-9cad-4d55-8f77-26b1ce2d2fbc
+---
+
+One of the biggest pain points during design-to-development handover is the loss of information or miscommunication between teams. Designers might create stunning designs, but without clear specifications, developers can struggle to implement them accurately. This often results in back-and-forth clarifications, missed deadlines, and frustration.
+
+<!--endintro-->
+
+Figma Dev Mode solves this issue by offering developers all the necessary design details in a developer-friendly format. With access to measurements, code snippets, and assets, developers can easily translate designs into code without needing constant input from designers.
```
Comment (danielmackay) [inline rules/use-figma-dev-mode/rule.md:18]: Can 'Figma Dev Mode' link to some documentation?

**Review by hveraus — APPROVED** [review]

File: rules/use-figma-dev-mode/rule.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+![Figure: Bad Example - A basic mockup or prototype with no technical details, forcing developers to guess measurements and styles.](figma-preview.png)  
+:::
+
+::: good  
+![Figure: Good Example - The design includes useful developer details such as color codes, font styles, measurements, and spacing.](figma-dev-mode.png)  
+:::
+
+### Watch this video on Figma Dev Mode
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/__ABPkb0aF8?si=gV7URLi-X5NiPs4y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (danielmackay) [inline rules/use-figma-dev-mode/rule.md:47]: YouTube videos can be embedded much simpler. Have a look at: https://www.ssw.com.au/rules/rule/#8-videos

File: rules/use-figma-dev-mode/rule.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+type: rule
+title: Do you use Figma Dev Mode to improve developer handovers?
+seoDescription: "Learn how Figma Dev Mode streamlines developer handovers by
+  providing a seamless bridge between design and development. type: rule"
+uri: use-figma-dev-mode
```
Comment (danielmackay) [inline rules/use-figma-dev-mode/rule.md:6]:
```suggestion
uri: figma-dev-mode
```

File: rules/use-figma-dev-mode/rule.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+authors:
+  - title: Betty Bondoc
+    url: https://www.ssw.com.au/people/betty-bondoc/
+created: 2024-10-30T14:20:35.753Z
+guid: 83be089e-9cad-4d55-8f77-26b1ce2d2fbc
+---
+
```
Comment (danielmackay) [inline rules/use-figma-dev-mode/rule.md:13]: This rule needs to be linked to a category. Do this by finding the markdown file for the category and adding the uri to this rule.

**Comment by tiagov8** [comment]
Hey @bettybondoc 

- [ ] 1. Please add it to a category

**Review by danielmackay — COMMENTED** [review]
Few comments

**Comment by tiagov8** [comment]
@hveraus  See my comments and the ones from Daniel... this was approved by you, but rule is missing some things.

**Comment by hveraus** [comment]
> @hveraus See my comments and the ones from Daniel... this was approved by you, but rule is missing some things.

sorry, I'm afraid I have to do a revert.

**Comment by bettybondoc** [comment]
@tiagov8 and @danielmackay comments addressed in https://github.com/SSWConsulting/SSW.Rules.Content/pull/9503/files

cc @hveraus

**Comment by tiagov8** [comment]
> > @hveraus See my comments and the ones from Daniel... this was approved by you, but rule is missing some things.
> 
> sorry, I'm afraid I have to do a revert.

@hveraus This was fixed on another PR. I think it was the better decision. Keep revert for things that could potentially break something... not the case :)

**Pull request merged**